internal/datanode/services.go

```diff
@@ -589,3 +594,19 @@ func (node *DataNode) DropCompactionPlan(ctx context.Context, req *datapb.DropCo
 	log.Ctx(ctx).Info("DropCompactionPlans success", zap.Int64("planID", req.GetPlanID()))
 	return merr.Success(), nil
 }
+
+func (node *DataNode) CheckHealth(ctx context.Context, req *milvuspb.CheckHealthRequest) (*milvuspb.CheckHealthResponse, error) {
```

Comment:
if we want to use it in serverless mode,should we check health only with one db? 
otherwise different tenant may affect each other

Reply:
We still use the cluster's health check results to check the health status of one database in the business layer. The next step will provide more detailed information about the cluster's health results, including the database, collection, and health status.